### ISSUE TITLE
Why do you need to multiply the result of final conv by 100?

### ISSUE THREAD

**Issue by AbnerCSZ** [fangchangma/self-supervised-depth-completion#20]
In the end of model.py
```

        if self.training:
            return 100 * y
        else:
            min_distance = 0.9
            return F.relu(100 * y - min_distance) + min_distance # the minimum range of Velodyne is around 3 feet ~= 0.9m
```
Could your tell me what is the 100 means?

**Comment by fangchangma** [comment]
> Could your tell me what is the 100 means?

It was legacy code and removing the multiplier should not affect the training much. However, the learning rate might require some adjustment.

**Comment by AbnerCSZ** [comment]
Thank you, by the way, why not add smooth loss in the dense d mode(args.w2 = 0 when mode is d). Have you compared the difference?

**Comment by fangchangma** [comment]
> Thank you, by the way, why not add smooth loss in the dense d mode(args.w2 = 0 when mode is d). Have you compared the difference?

I believe I did the experiment but did not observe improvement. Please let me know if you observe otherwise.
